pydis_site/apps/content/models/tag.py

```diff
@@ -0,0 +1,75 @@
+import collections.abc
+import json
+
+from django.db import models
+
+
+class Commit(models.Model):
+    """A git commit."""
+
+    URL_BASE = "https://github.com/python-discord/bot/commit/"
+
+    sha = models.CharField(
+        help_text="The SHA hash of this commit.",
+        primary_key=True,
+        max_length=40,
+    )
+    message = models.TextField(help_text="The commit message.")
+    date = models.DateTimeField(help_text="The date and time the commit was created.")
+    author = models.TextField(help_text="The person(s) who created the commit.")
+
+    @property
+    def url(self) -> str:
+        """The URL to the commit on GitHub."""
+        return self.URL_BASE + self.sha
+
+    def lines(self) -> collections.abc.Iterable[str]:
+        """Return each line in the commit message."""
+        for line in self.message.split("\n"):
+            yield line
+
+    def format_users(self) -> collections.abc.Iterable[str]:
+        """Return a nice representation of the user(s)' name and email."""
+        for author in json.loads(self.author):
+            yield f"{author['name']} <{author['email']}>"
+
+
+class Tag(models.Model):
+    """A tag from the python-discord server."""
```

Comment:
Since we're hard coding this to the bot repo, should this instead be
```suggestion
    """A tag from the python-discord bot repository."""
```